rest/blip_sync.go

```diff
@@ -754,8 +783,16 @@ func (bh *blipHandler) sendRevisionWithProperties(body db.Body, sender *blip.Sen
 	if len(attDigests) > 0 {
 		// Allow client to download attachments in 'atts', but only while pulling this rev
 		bh.addAllowedAttachments(attDigests)
-		sender.Send(outrq.Message)
+		if !sender.Send(outrq.Message) {
+			bh.Logf(base.LevelDebug, base.KeySync, "Couldn't send rev message to client. Closed connection? %s", outrq)
+			return
```

Comment:
As discussed, add an error return to this that can be used to terminate the changes loop.